README.md

```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## **About Me**
+
+Hello, My name is **Tianhong Yin**, an MA student in international relations at the University of Chicago.
+
+My **research interests** include:
+* Conflict Studies
+* American Foreign Policy
+* American Politics
+* Quantitative Methodology
+
+I am a cat person and I love to know everything about cats during my spare time. Here is the [link to a cat forum](http://thecatsite.com) I often visit to learn about cats.
+
+I also have two cats in my home back in China, and below is their photo.
```

Comment:
Another thing you could think about is resizing/cropping the image. I am not sure how to do that, though, other than cropping/resizing the source (I tried to do this in my own assignment.)